python/paddle/distributed/launch.py

```diff
@@ -11,100 +11,54 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+paddle.distributed.launch is a module that spawns multiple distributed 
+process on each trainning node for gpu trainning.
+
+Usage:
+    In both of single node training or multiple node training, this module 
+launch a process on each of the given gpu card.
+    
+    1. for single node four cards training
+       python -m torch.distributed.launch --selected_gpus="0,1,2,3,4" \
```

Comment:
i think it better to indicate that "selected_gpus" option is alternative in this example, or show an example with minimum options explicitly.

Reply:
so as to illustrate the convenience of our toolkit

Reply:
Done.